### DISCUSSION TITLE
Defined bodylevel e.g. for thermostats

### DISCUSSION THREAD

**Discussion by MubiTec** [Zefau/ioBroker.jarvis#1155] [Q&A]
I have a thermostat in configuration like this

https://www.smarthome-tricks.de/software-iobroker/iobroker-jarvis-teil-16-heizkoerperthermostat/amp/

So as I only need temperatures to set from 17-25°C is this possible to show only this area and not from 0-100?
I tried it with min max configuration in io Riker data point but without success.

**Comment by Zefau** [comment]
Change your function to `heating` should work when the state is called `level`. Then default settings apply.

With any other function or state you simply use min/max in device properties.

Which version are you on?

**Reply by MubiTec** [reply]
OK, thank you. I will try this afternoon.
I'm using 3.0.0 112
